src/code/z_face_reaction.c

```diff
@@ -1,3 +1,176 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_face_reaction/func_800F1250.s")
+/**
+ * Indexes of this array:
+ * - index 0x00: PLAYER_MASK_MASK_OF_TRUTH
+ * - index 0x01: PLAYER_MASK_KAFEIS_MASK
+ * - index 0x02: PLAYER_MASK_ALL_NIGHT_MASK
+ * - index 0x03: PLAYER_MASK_BUNNY_HOOD
+ * - index 0x04: PLAYER_MASK_KEATON_MASK
+ * - index 0x05: PLAYER_MASK_GAROS_MASK
+ * - index 0x06: PLAYER_MASK_ROMANIS_MASK
+ * - index 0x07: PLAYER_MASK_CIRCUS_LEADERS_MASK
+ * - index 0x08: PLAYER_MASK_POSTMANS_HAT
+ * - index 0x09: PLAYER_MASK_COUPLES_MASK
+ * - index 0x0A: PLAYER_MASK_GREAT_FAIRYS_MASK
+ * - index 0x0B: PLAYER_MASK_GIBDO_MASK
+ * - index 0x0C: PLAYER_MASK_DON_GEROS_MASK
+ * - index 0x0D: PLAYER_MASK_KAMAROS_MASK
+ * - index 0x0E: PLAYER_MASK_CAPTAINS_HAT
+ * - index 0x0F: PLAYER_MASK_STONE_MASK
+ * - index 0x10: PLAYER_MASK_BREMEN_MASK
+ * - index 0x11: PLAYER_MASK_BLAST_MASK
+ * - index 0x12: PLAYER_MASK_MASK_OF_SCENTS
+ */
+u16 sReactionTextIds[][PLAYER_MASK_GIANTS_MASK - 1] = {
```

Comment:
Any reason why this isn't declared as "static" but still has an "s" prefix to the variable name?

Reply:
Because I forgot to add it :sweat_smile: